unattended_installer/common_functions/common.sh

```diff
@@ -68,39 +68,39 @@ function common_checkInstalled() {
     dashboard_installed=""
 
     if [ "${sys_type}" == "yum" ]; then
-        wazuh_installed=$(yum list installed 2>/dev/null | grep wazuh-manager)
+        wazuh_installed=$(yum list installed 2>/dev/null | grep wazuh-manager | sed 's/  */ /g'| cut -d' ' -f2 | sed "s/-.*//g")
     elif [ "${sys_type}" == "apt-get" ]; then
-        wazuh_installed=$(apt list --installed  2>/dev/null | grep wazuh-manager)
+        wazuh_installed=$(apt list --installed  2>/dev/null | grep wazuh-manager | cut -d' ' -f2 | sed -e "s/-.*//")
     fi
 
     if [ -d "/var/ossec" ]; then
         wazuh_remaining_files=1
     fi
 
     if [ "${sys_type}" == "yum" ]; then
-        indexer_installed=$(yum list installed 2>/dev/null | grep wazuh-indexer)
+        indexer_installed=$(yum list installed 2>/dev/null | grep wazuh-indexer | sed 's/  */ /g'| cut -d' ' -f2 | sed "s/-.*//g")
     elif [ "${sys_type}" == "apt-get" ]; then
-        indexer_installed=$(apt list --installed 2>/dev/null | grep wazuh-indexer)
+        indexer_installed=$(apt list --installed 2>/dev/null | grep wazuh-indexer | cut -d' ' -f2 | sed -e "s/-.*//")
```

Comment:
Two lines before you call command `sed "s/-.*//g"`, without using argument `-e`. What is the reason for the inconsistency?

Reply:
Fixed [c32120b](https://github.com/wazuh/wazuh-packages/pull/1771/commits/c32120bbbe56fc8c26a39616fe2d02e573a81565)